packages/terra-form-select/tests/jest/SelectField.test.jsx

```diff
@@ -18,6 +18,20 @@ it('should render a default SelectField component', () => {
   expect(shallowWithIntl(select)).toMatchSnapshot();
 });
 
+it('should render a disabled SelectField component', () => {
+  const select = (
+    <SelectField disabled label="Label" defaultValue="blue" selectId="select-id">
+      <SelectField.Option value="blue" display="Blue" />
+      <SelectField.Option value="green" display="Green" />
+      <SelectField.Option value="purple" display="Purple" />
+      <SelectField.Option value="red" display="Red" />
+      <SelectField.Option value="violet" display="Violet" />
+    </SelectField>
+  );
+
+  expect(shallowWithIntl(select)).toMatchSnapshot();
+});
```

Comment:
We need to add another jest test to verify the `selectAttrs.disabled` also renders a disabled input field